recipes/crowcpp-crow/all/conanfile.py

```diff
@@ -16,13 +17,15 @@ def _source_subfolder(self):
 
     def requirements(self):
         self.requires("boost/1.75.0")
-        '''To be removed in next released versions'''
+        # To be removed in next released versions.
```

Comment:
Someone wrote this, have no idea what they meant... And *when* they meant it. 
I was just doing the update...

Reply:
https://github.com/conan-io/conan-center-index/pull/4085/files#r553689508

Reply:
openssl is nowhere to be found in the current cmake script.
https://github.com/CrowCpp/crow/blob/master/CMakeLists.txt
So I think `openssl` can be completely removed.

Reply:
It's needed for older versions, so how do we compromise on that?

(And I though it was a 4 line PR to make someone in the Issues tab happy hahah)

Reply:
Check if the fix is good enough.

Reply:
I feel your pain.
There does not exists such a thing as an easy pr.